### PR TITLE
Resolve location error

### PR DESCRIPTION
**Pull request by markscholman**
### Description of the change

When using the [resourceGroup().location] I got an error about that the region is not known. When hardcoding or parameterize these it worked.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@markscholman__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by krnese** [comment]
@markscholman there's another "location" within the properties envelope that refers to resourceGroup().location. can you add the parameter to this too.

File: app-service-environment-serviceplan-logicapp/azuredeploy.json [hunk 1]
```diff
@@ -8,6 +8,12 @@
         "description": "The name to use for the new AppService Environment"
       }
     },
+    "location": {
```
Comment (bmoore-msft) [inline app-service-environment-serviceplan-logicapp/azuredeploy.json]: using resourceGroup().location is a better practice here, is there something specific we need to break it out into a param for?

Comment (markscholman) [inline app-service-environment-serviceplan-logicapp/azuredeploy.json]: When doing so deployment fails. That's why I changed it back to a parameter and commit it. Maybe a fix is needed in the parsing of the template in order to support this.

Comment (bmoore-msft) [inline app-service-environment-serviceplan-logicapp/azuredeploy.json]: I'm not able to get a successful deployment with this template regardless... it looks like the RP does have a bug in handling locations, but even when I get around that the deployment fails on finding the vnet.  How do we fix that?

Comment (markscholman) [inline app-service-environment-serviceplan-logicapp/azuredeploy.json]: If i recall correctly the network should be already in place and is used as a resource reference. (What can be outside the RG you deploy this template to). The subnet should be a dedicated worker subnet only no other vm's should live there.

Comment (bmoore-msft) [inline app-service-environment-serviceplan-logicapp/azuredeploy.json]: Yeah I tried a vnet in the RG and outside the RG (in the same region) but never got a successful deployment of this template.

**Comment by bmoore-msft** [comment]
@yossidahan - I've been unable to get any variation of this sample to successfully deploy, even with a classic vnet (deployment times out).  Can you update the sample and provide instructions to make it work?

**Comment by azurecla** [comment]
__@markscholman__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Pull request closed without merging**